src/services/BackendService.js

```diff
@@ -15,14 +15,12 @@
  */
 import { SERVER_URL } from "@/services/service.const";
 import axios from "axios";
-import { inject } from "vue";
+
 export default class BackendService {
-  async getPassportV1(assetId) {
-    return await this.getPassport("v1", assetId);
+  async getPassportV1(assetId, jwtToken) {
```

Comment:
This bug is related with the authentication token, it was not able to inject vue in a JS component so it gave always a exception and gave timeout. Fixed by giving token from outside.